pegasus/sites.v3/code.org/public/teacher-dashboard/review-hociyskvuwa/splat.haml

```diff
@@ -98,7 +89,7 @@ title: Review Hour of Code special events
 
   function submitReview(secret, review, row) {
     $.ajax({
-      url: "/v2/forms/HocSignup2017/" + secret + "/review",
+      url: "/v2/forms/HocSignup2018/" + secret + "/review",
```

Comment:
We use DCDO `hoc_year` so we don't have to remember all the places to update. :)

https://github.com/code-dot-org/code-dot-org/blob/0cd6428b533faa05ac4c95d89bf09ffd4bb6aba0/pegasus/sites.v3/hourofcode.com/public/events/organizations.haml#L8